fjord/manage_utils.py

```diff
@@ -63,6 +61,9 @@ def check_dependencies():
     from pip.req import parse_requirements
     from pip.download import PipSession
 
+    # Avoid circular import
+    from fjord import path
```

Comment:
`path` is such a goofy thing. It pains me to have circular imports for something that's such a trivial and non-interesting bit of code. We should figure out what to do with `ROOT` and `path`.